sdk/agent/events/meta.go

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package events
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/google/uuid"
+	sdkGRPC "github.com/nginx/agent/sdk/v2/grpc"
+	"github.com/nginx/agent/sdk/v2/proto"
+	commonProto "github.com/nginx/agent/sdk/v2/proto/common"
+	eventsProto "github.com/nginx/agent/sdk/v2/proto/events"
+)
+
+type AgentEventMeta struct {
+	module        string
+	version       string
+	pid           string
+	message       string
+	hostname      string
+	systemUuid    string
+	instanceGroup string
+	tags          []string
+}
+
+func NewAgentEventMeta(
+	module, version, pid, message, hostname, systemUuid, instanceGroup string,
+	tags []string,
+) *AgentEventMeta {
+	return &AgentEventMeta{
+		module:        module,
+		version:       version,
+		pid:           pid,
+		message:       message,
```

Comment:
No need for a message field as it should always be `nginx-agent`

Reply:
in the Agent but is someone uses the SDK they can send their own messages